### PR TITLE
City Gen version 2

### PR DESCRIPTION
**Pull request by tkwebster**
1. Set the title to exactly what you are adding.

2. Which team is this for?
Gameplay

3. Give a short description of what you are implementing:

- Building Walls no longer overlap
- Invisible Walls added
- Wall colors added as a test feature
- City Parameters are now synced, allowing for the parameters to be chosen by host in future

### CODE REVIEW

File: Local War/Assets/Resources/Script/CityGen/MeshGenerator.cs [hunk 1]
```diff
@@ -183,22 +188,87 @@ void BuildWall(float px, float py, float pz, float sx, float sy, float sz)
...
+			cubeRenderer.material.SetColor("_Color", yellow);
+		}
+		else if (choice >= 7) 
+		{
+			cubeRenderer.material.SetColor("_Color", red);
+		}
+		else 
+		{
+			cubeRenderer.material.SetColor("_Color", teal);
+		}
```
Comment (JasonStallkamp) [inline Local War/Assets/Resources/Script/CityGen/MeshGenerator.cs]: Indentation is not consistent

Comment (JasonStallkamp) [reply Local War/Assets/Resources/Script/CityGen/MeshGenerator.cs]: Other than that it is good to merge

**Review by JasonStallkamp — CHANGES_REQUESTED** [review]
Fix indentation Local War/Assets/Resources/Script/CityGen/MeshGenerator.cs on line 191 to 208

**Comment by tkwebster** [comment]
What's inconsistent in the indentation? The if statements? I just opened it up again and it looks correct to me.

File: Local War/Assets/Resources/Script/CityGen/MeshGenerator.cs (hunk 1 above)
Comment (JasonStallkamp) [inline Local War/Assets/Resources/Script/CityGen/MeshGenerator.cs]: Here is the code with an extra indent

Comment (JasonStallkamp) [reply Local War/Assets/Resources/Script/CityGen/MeshGenerator.cs]: Could not find the indentation in the unity editor

**Review by JasonStallkamp — APPROVED** [review]

**Pull request merged**